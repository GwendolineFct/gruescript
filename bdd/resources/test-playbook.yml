

---


  # this is a test playbook to check things get migrated correctly
  - hosts: servers
    vars:
      myvar: 'Foo bar'
      anothervar: "spam & eggs"   # what about a shruberry ?
      longtext: |
        We can have multiple
        lines strings in YAML
        by doing so
    pre_tasks:
      - name: My first pre task
        obsolete:
          param1: I don't exist, what can I do ?
        when: false
      # don't forget to comment your code
      - name: another pre task
        freeform: Oh, freedom (freedom), freedom (freedom)
        become: true

        # else it's hard to understand
    role:
      - {role: test}
    tasks:
      - name: a simple task
        realfacts:
    post_tasks:
      - name: My first post task
        breaking:
          param2: Don't go breaking my heart
      - name: another post task
        registration:
        register: somevar
  # This is the end my friend