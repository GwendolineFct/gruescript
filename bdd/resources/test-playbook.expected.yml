

---


  # this is a test playbook to check things get migrated correctly
  - hosts: servers
    vars:
      myvar: 'Foo bar'
      anothervar: "spam & eggs"   # what about a shruberry ?
      longtext: |
        We can have multiple
        lines strings in YAML
        by doing so
    pre_tasks:
      - name: My first pre task
# *** MIG ***  ERROR : module `obsolete` does not exist in latest version
        obsolete:
          param1: I don't exist, what can I do ?
        when: false
      # don't forget to comment your code
      - name: another pre task
# *** MIG ***  WARNING : Cannot perform migration checks on free-form parameters
        migration.test.freeform: Oh, freedom (freedom), freedom (freedom)
        become: true

        # else it's hard to understand
    role:
      - {role: test}
    tasks:
# *** MIG ***  WARNING : There is a breaking change in the facts returned.
# Please check documentation for latest version of module `migration.test.realfacts`
# https://docs.ansible.com/ansible/latest/collections/migration/test/realfacts_module.html
      - name: a simple task
        migration.test.realfacts:
    post_tasks:
      - name: My first post task
# *** MIG ***  ERROR : missing parameter `param1` is required in latest version
        migration.test.breaking:
# *** MIG ***  ERROR : unknown module parameter `param2` in latest version
          param2: Don't go breaking my heart
      - name: another post task
        migration.test.registration:
# *** MIG ***  WARNING : There is a breaking change in the returned values
        register: somevar
  # This is the end my friend